src/main/java/com/radixdlt/crypto/ECKeyPair.java

```diff
@@ -42,6 +44,7 @@
 /**
  * Asymmetric EC key pair provider fixed to curve 'secp256k1'.
  */
+@SecurityCritical({ SecurityKind.KEY_GENERATION })
```

Comment:
Hmm I think this class should be flagged with `SecurityKind .Signing` as well, even though it is forwarding calls to `ECKeyUtils.keyHandler.sign`, it can still do so incorrectly, do you agree?

Reply:
Hmm an possibly also `PK_ENCRYPT` and `PK_DECRYPT`? It is forwarding to `return this.publicKey.encrypt(data);` and `return this.publicKey.decrypt(data);`... Not as sure about those two though.

Reply:
As this class doesn't actually do signing or encryption, I wasn't going to include them here, but maybe I should.

Will include.